screwdriver/screwdriver.go

```diff
@@ -41,11 +42,31 @@ type Step struct {
 	Command string `json:"command"`
 }
 
+// MapSlice is an associative array which preserves order
+type MapSlice struct {
+	Body []map[string]string
```

Comment:
it should be sequence of `struct { Key, Value string }`, because this map contains single key/value only

Reply:
I fixed it.